### DISCUSSION TITLE
Slack bot improvements for Github Discussions

### DISCUSSION THREAD

**Discussion by LucaColonnello** [community/community#29061] [Discussions]
At Trainline, we use Github Discussions to bring together our communities of developers. This helps them support each other with common issues, but also drive our engineering processes, tooling and tech stack forward thanks to everybody’s contribution using polls and general discussions.

This is great and we love the product!
However, using the slack bot to subscribe to discussions, is quite a limited experience.

We have channels that subscribe to specific discussion platforms, but at times this can create some noise and people mute those channels.

It would be super useful if there was a way to subscribe based on labels and categories for discussions! This is possible for issues already, so it would be great if it was available for discussions too!

**Comment by alansill** [comment]
It's available now to add discussions, but only for creation of marking as an answer. The Slack integration does not allow creation of a channel to monitor discussions, for example. 
https://github.com/orgs/community/discussions/2844#discussioncomment-1852574
